src/lib/services/transaction-service.ts

```diff
@@ -334,7 +335,7 @@ export async function createSafeTransaction(
     sender: ethers.utils.getAddress(input.sender), // Get the checksummed address
     confirmationsRequired: threshold,
     safeTxGas,
-    nonce,
+    nonce: nonce + 1, // We got the latest transaction, so add 1 to it.
```

Comment:
how does gnosis create queued transactions, I though they created transactions without nonce?

Reply:
It uses whatever the provided nonce value is, and the nonce value is required.

Reply:
but if there 2 queued transactions get created they will have conflicting nonce, no?

Reply:
That fetched nonce value is based on queued transactions, not whatever the active transaction is